s3torchconnector/src/s3torchconnector/lightning/s3_lightning_checkpoint.py

```diff
@@ -3,20 +3,23 @@
 
 from typing import Optional, Dict, Any
 
+import lightning
 import torch
 
 from lightning.pytorch.plugins.io import CheckpointIO
 
 from .._s3client import S3Client
 from .._s3dataset_common import parse_s3_uri
+from .._user_agent import UserAgent
 
 
 class S3LightningCheckpoint(CheckpointIO):
     """A checkpoint manager for S3 using the :class:`CheckpointIO` interface."""
 
     def __init__(self, region: str):
         self.region = region
-        self._client = S3Client(region)
+        user_agent = UserAgent(["lightning", lightning.__version__])
```

Comment:
Do we want to pass in `"lightning"` here, or `S3LightningCheckpoint`?

Reply:
This is consistent with the format we have now.